### PR TITLE
Added Nextcloud 26 upgrade changes docs

### PR DESCRIPTION
**Pull request by andrey18106**
Task from @DaphneMuller

### CODE REVIEW

**Review by bigcat88 — APPROVED** [review]
Good

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst [hunk 1]
```diff
@@ -57,9 +69,50 @@ App container parameters with pascal case names ``AppName``, ``UserId`` and ``We
...
+* New ``OCP\BeforeSabrePubliclyLoadedEvent`` event dispatched on public webdav endpoints (it can be used just like ``OCP\SabrePluginEvent`` to inject additional Sabre plugins in apps for example) (`PR#35789 <https://github.com/nextcloud/server/pull/35789>`_)
+
+Deprecated
+----------
+
+* ``OCP\BackgroundJob\IJobList::getJobs`` method was deprecated (`PR#36073 <https://github.com/nextcloud/server/pull/36073>`_)
+* Controller action annotation ``@UseSession`` is deprecated. Use new ``UseSession`` attribute instead (`PR#36363 <https://github.com/nextcloud/server/pull/36363>`_ by `ChristophWurst <https://github.com/ChristophWurst>`_)
+* **Notifications jQuery event deprecated**: The ``OCA.Notification.Action`` event of the notifications app is deprecated in favor of a ``notifications:action:executed`` event-bus event with (`PR#728 <https://github.com/nextcloud/notifications/pull/728>`_ by `nickvergessen <https://github.com/nickvergessen>`_)
+
+Changes for admins
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:110]: wrong manual. these changes should go into the admin_manual

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst [hunk 2]
```diff
@@ -57,9 +69,50 @@ App container parameters with pascal case names ``AppName``, ``UserId`` and ``We
...
+* ``Auto-Submitted`` header for emails: There now is a new method on the ``OCP\Mail\IMessage`` interface ``IMessage::setAutoSubmitted()``. With this method you can specify that an email was an automatic email or response, to allow mail servers to better detect if an out-of-office reply should be sent, better store/filter the emails and so on. Possible values are documented in the ``OCP\Mail\Headers\AutoSubmitted`` interface. (`PR#36033 <https://github.com/nextcloud/server/pull/36033>`_ by `bennet0496 <https://github.com/bennet0496>`_)
+* ``OCP\BackgroundJob\IJobList::getJobsIterator`` method was added (`PR#36073 <https://github.com/nextcloud/server/pull/36073>`_)
+* New ``OCP\BeforeSabrePubliclyLoadedEvent`` event dispatched on public webdav endpoints (it can be used just like ``OCP\SabrePluginEvent`` to inject additional Sabre plugins in apps for example) (`PR#35789 <https://github.com/nextcloud/server/pull/35789>`_)
```
Comment (ChristophWurst) [inline developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:101]: documentation on the events page is missing

Comment (ChristophWurst) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:101]: exactly :)

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst (hunk 1 above)
Comment (bigcat88) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:110]: Where we should put it there? I see only one page with changes [Maintenance and release schedule](https://docs.nextcloud.com/server/stable/admin_manual/release_schedule.html) and it itself links to develop manual.

File: developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst (hunk 2 above)
Comment (bigcat88) [reply developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst:101]: thanks, we will do it today. _to clarify: we received task only to update one page, so we did what we were asked, didn't know that we should update more info_ :)

**Pull request merged**